todo-app/src/ListItem.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import './App.css';
+
+ export default class ListItem extends React.Component {
+   constructor(props) {
+    super(props);
+    this.state = {
+      showEdit: true,
+      editToDo: {
+        description: props.description,
+        done: false
+      }
+    }
+
+  }
+
+  Edit(showEdit) {
```

Comment:
Why did you use a capital letter here? This is just a function of the `ListItem` component. If you meant for this to be another component, then you should put it in another file like `Edit.js` and import it and use it here.